### PR TITLE
Add display settings

### PR DESCRIPTION
**Pull request by Moltenship**
Currently, we getting cookies from `getInitialProps` in order to fix hydration error.

`getInitialProps` in `_app.tsx` can be omitted once we move to `app` dir architecture.

### CODE REVIEW

File: src/features/search/components/CharacterSearch/CharacterSearch.tsx [hunk 1]
```diff
@@ -8,6 +8,10 @@ import { Pagination } from 'src/api/models/pagination';
 import { useForm, useWatch } from 'react-hook-form';
 import { Form } from 'src/components/Form/Form';
 import { useDebounce } from 'usehooks-ts';
+import { CensorService } from 'src/api/services/censorService';
+
+// import { useSettings } from 'src/hooks/useSettings';
```
Comment (moetazaneta) [inline src/features/search/components/CharacterSearch/CharacterSearch.tsx]: Remove dead code

File: src/features/search/components/CharacterSearchForm/CharacterSearchForm.tsx [hunk 2]
```diff
@@ -15,6 +15,10 @@ import { IconButton } from 'src/components/IconButton/IconButton';
 import { Select } from 'src/components/Select';
 import { TextInput } from 'src/components/TextInput/TextInput';
 import { useDebounce } from 'usehooks-ts';
+import { CensorService } from 'src/api/services/censorService';
+
+// import { useSettings } from 'src/hooks/useSettings';
```
Comment (moetazaneta) [inline src/features/search/components/CharacterSearchForm/CharacterSearchForm.tsx]: Remove dead code

File: src/features/search/components/VnSearch/VnSearch.tsx [hunk 3]
```diff
@@ -8,6 +8,10 @@ import { CardSkeleton } from 'src/components/Card/CardSkeleton';
 import { Form } from 'src/components/Form/Form';
 import { ChildrenMultiplier } from 'src/components/ChildrenMultiplier/ChildrenMultiplier';
 import { Pagination } from 'src/api/models/pagination';
+import { CensorService } from 'src/api/services/censorService';
+
+// import { useSettings } from 'src/hooks/useSettings';
```
Comment (moetazaneta) [inline src/features/search/components/VnSearch/VnSearch.tsx]: Remove dead code

File: src/features/search/components/VnSearchForm/VnSearchForm.tsx [hunk 4]
```diff
@@ -18,6 +18,10 @@ import { Select } from 'src/components/Select';
 import { Slider } from 'src/components/Slider/Slider';
 import { TextInput } from 'src/components/TextInput/TextInput';
 import { useDebounce } from 'usehooks-ts';
+import { CensorService } from 'src/api/services/censorService';
+
+// import { useSettings } from 'src/hooks/useSettings';
```
Comment (moetazaneta) [inline src/features/search/components/VnSearchForm/VnSearchForm.tsx]: Remove dead code

**Review by moetazaneta — APPROVED** [review]

File: src/features/search/components/CharacterSearch/CharacterSearch.tsx [hunk 5]
```diff
@@ -9,7 +9,9 @@ import { useForm, useWatch } from 'react-hook-form';
 import { Form } from 'src/components/Form/Form';
 import { useDebounce } from 'usehooks-ts';
 import { CensorService } from 'src/api/services/censorService';
-import { useSettings } from 'src/hooks/useSettings';
+
+// import { useSettings } from 'src/hooks/useSettings';
```
Comment (NightOfNights) [inline src/features/search/components/CharacterSearch/CharacterSearch.tsx]: Seems like dead code is still there

**Review by NightOfNights — APPROVED** [review]

**Pull request merged**